ddtrace/contrib/internal/celery/signals.py

```diff
@@ -181,9 +183,21 @@ def trace_after_publish(*args, **kwargs):
     if span is None:
         return
     else:
-        nodename = span.get_tag("celery.hostname")
-        if nodename is not None:
-            _, host = nodenames.nodesplit(nodename)
+        broker_url = current_app.conf.broker_url
+
+        if broker_url == "memory://":
```

Comment:
Is it possible for this to be some other broker, ie: `redis://` going by https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html#brokers ?

If so, should this just be anything ending with `://` as the string: @djova and @wconti27 ?

Reply:
It would be the schema + full url for anything that's not `memory`. So we should be fine.